lib/vauth/vauth.c

```diff
@@ -53,6 +53,25 @@
  *
  * Returns a pointer to the newly allocated SPN.
  */
+
+#if defined(USE_OPENSSL)
+char *Curl_auth_build_spn_openssl(const char *service, const char *host,
```

Comment:
It is called in digest.c.

Reply:
Hi @bagder, can you share me the full list of coding style changes expected?

When I do "**make checksrc**" on Windows, I am getting the following message:

```
make checksrc
make: *** No rule to make target `checksrc'.  Stop.
```

Reply:
Hi @bagder, I have re-written the function so that the Windows SSPI and SCHANNEL macros call one logic for SPN while the other mechanisms call the other SPN. I think it looks better now and avoids the confusion you had earlier.